lib/sorcery/controller/submodules/session_timeout.rb

```diff
@@ -52,6 +52,7 @@ def register_login_time(_user, _credentials = nil)
           def validate_session
             session_to_use = Config.session_timeout_from_last_action ? session[:last_action_time] : session[:login_time]
             if (session_to_use && sorcery_session_expired?(session_to_use.to_time)) || sorcery_session_invalidated?
+              forget_me! if current_user.present? && Config.submodules.include?(:remember_me)
```

Comment:
Wouldn't it make more sense to just call `logout` here?
There might be other before/after logout hooks configured and `logout` already takes care of calling all of those.
Just discovered this PR after I ran into the same problem and opened my PR with `logout` here:
https://github.com/Sorcery/sorcery/pull/358

Reply:
Hello @jankoegel i saw you PR, yes is better then mine. Makes more sense to call logout. Can i add the tests in your PR?

Reply:
Sure, I invited you as a collaborator to my fork. Let me know if that works for you & thanks for preparing the specs!